QEfficient/utils/_utils.py

```diff
@@ -61,12 +60,21 @@ def hf_download(
             retry_count += 1
 
         except HTTPError as e:
-            retry_count = max_retries
+            retry_count = Constants.MAX_RETRIES
```

Comment:
I think its better to use  `break` statement instead of `retry_count = Constants.MAX_RETRIES`?

Reply:
We should raise the error here, if we only break, the execution will continue even when the model is not downloaded, and we will get `undefined variable model_path` as error.
Please change this to
```
if e.response.status_code == 401:
    logger.info("You need to pass a valid `--hf_token=...` to download private checkpoints.")
raise e
```